pkg/management/http.go

```diff
@@ -31,21 +33,53 @@ func MakeManagementHandler(e endpoint.Endpoint, logger log.Logger) *http_transpo
 func decodeManagementRequest(_ context.Context, req *http.Request) (interface{}, error) {
 	var request = map[string]string{}
 
-	// Fetch path parameter such as realm, userID, ...
+	// Fetch and validate path parameter such as realm, userID, ...
+	var pathParams = map[string]string{
+		"realm":        "^[a-zA-Z0-9_-]{1,36}$",
+		"userID":       "^[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}$",
+		"clientID":     "^[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}$",
+		"roleID":       "^[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}$",
+		"credentialID": "^[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}$",
```

Comment:
You could use the "private" constant you created in api (of course, making it "public")